Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/BaseTeleOp.java

```diff
@@ -1,11 +1,42 @@
 package org.firstinspires.ftc.team417_PowerPlay;
 
 abstract public class BaseTeleOp extends BaseOpMode{
+    boolean grabberIsOpen = false;
+    double armPower = 0.0;
+
     public void driveUsingControllers() {
-        double x = gamepad1.left_stick_x;
-        double y = - gamepad1.left_stick_y;
-        double turning = gamepad1.right_stick_x;
+        double x = gamepad1.left_stick_x * 0.5;
+        double y = -gamepad1.left_stick_y * 0.5;
+        double turning = gamepad1.right_stick_x * 0.5;
```

Comment:
Our drivers felt that they wouldn't need the speed since our robot does go really fast, for better control they wanted to just permanently cap the speed

Reply:
In that case, can you make this a constant so you can adjust it easily later?